### DISCUSSION TITLE
Nord Theme

### DISCUSSION THREAD

**Discussion by en3r0** [TriliumNext/Trilium#1930] [Show and tell]
It could use some tweaks, but I am happy with it for now.

https://github.com/en3r0/Trilium-Nord-Theme

![image](https://user-images.githubusercontent.com/7061425/117723591-8e9dd180-b1b0-11eb-8136-3ae053c10858.png)

**Comment by TheOneValen** [comment]
It does not work in v0.57.5 :(

**Comment by en3r0** [comment]
I am using it on v0.57.5 with no issue. Can you be more specific?

**Reply by TheOneValen** [reply]
I copy the css from the repo to css note. Put the attribute there. Choose the theme in options -> it shows the default light theme :(

**Reply by en3r0** [reply]
Are you following the custom CSS in the wiki? https://github.com/zadam/trilium/wiki/Themes#custom-css-themes

**Reply by TheOneValen** [reply]
What do you mean? I got the file from your repo. I setup other themes without issues, if you mean the attribute stuff. This is the only one (as of now) I ran into problems. And Nord is my favorite theme in vscode 😄

**Reply by en3r0** [reply]
Just to check, did is the note type text/css ? Also, did you set the attribute to #appTheme=Nord ?
